### PR TITLE
Angular: Clean and improve angular-cli examples

### PR DESCRIPTION
**Pull request by ThibaudAV**
Issue: N/A

## What I did

- Moves existing story examples into a structure (Core, Basics, Addons, Legacy, Others)
- Replace Knobs by Args + controls in examples 
- Replace "storiesOf" by "CSF"

Don't hesitate if you have other ideas or suggestions :) 

Final result in storybook menu
image

## How to test

- Is this testable with Jest or Chromatic screenshots? N/A
- Does this need a new example in the kitchen sink apps? N/A
- Does this need an update to the documentation? N/A

image

### CODE REVIEW

**Comment by ThibaudAV** [comment]
aaaaa
it's not very easy to do code review by commit because of the storyshot. If you want I can remove them from each commit and do a single update commit at the end. 🤷‍♂️

File: examples/angular-cli/src/stories/addons/READEME.stories.mdx [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import { Meta } from '@storybook/addon-docs/blocks';
```
Comment (gaetanmaisse) [inline examples/angular-cli/src/stories/addons/READEME.stories.mdx:1]: There is a typo in the file name: `READEME` -> `README`

File: examples/angular-cli/src/stories/basics/READEME.stories.mdx [hunk 2]
```diff
@@ -0,0 +1,7 @@
+import { Meta } from '@storybook/addon-docs/blocks';
```
Comment (gaetanmaisse) [inline examples/angular-cli/src/stories/basics/READEME.stories.mdx:1]: Same here

File: examples/angular-cli/src/stories/core/READEME.stories.mdx [hunk 3]
```diff
@@ -0,0 +1,7 @@
+import { Meta } from '@storybook/addon-docs/blocks';
```
Comment (gaetanmaisse) [inline examples/angular-cli/src/stories/core/READEME.stories.mdx:1]: Same here

File: examples/angular-cli/src/stories/legacy/READEME.stories.mdx [hunk 4]
```diff
@@ -0,0 +1,7 @@
+import { Meta } from '@storybook/addon-docs/blocks';
```
Comment (gaetanmaisse) [inline examples/angular-cli/src/stories/legacy/READEME.stories.mdx:1]: and here

File: examples/angular-cli/src/stories/welcome-angular.stories.ts [hunk 5]
```diff
@@ -0,0 +1,14 @@
+import { Story, Meta } from '@storybook/angular';
+import { linkTo } from '@storybook/addon-links';
+import { AppComponent } from '../app/app.component';
+
+export default {
+  title: 'Welcome/ To Angular',
```
Comment (gaetanmaisse) [inline examples/angular-cli/src/stories/welcome-angular.stories.ts:6]: Missing space after `Welcome` to be consistent with the other files

File: examples/angular-cli/src/stories/welcome-storybook.stories.ts [hunk 6]
```diff
@@ -0,0 +1,14 @@
+import { Story, Meta } from '@storybook/angular';
+import { Welcome } from '@storybook/angular/demo';
+import { linkTo } from '@storybook/addon-links';
+
+export default {
+  title: 'Welcome/ To Storybook',
```
Comment (gaetanmaisse) [inline examples/angular-cli/src/stories/welcome-storybook.stories.ts:6]: Missing space after `Welcome` to be consistent with the other files

File: examples/angular-cli/src/stories/welcome-angular.stories.ts (hunk 5 above)
Comment (ThibaudAV) [reply examples/angular-cli/src/stories/welcome-angular.stories.ts:6]: not missing, I did it on purpose.

Personally I prefer with spaces. I find it more readable. 


but there is a strange behavior with `storySort`

If all the story have a space before the name, I have to take it into account in `storySort.order`
If part of the stories have a space. It groups well together. but I don't have to add space in `storySort.order`
and if none of them have space, no problem

I was expecting that the stroybook would not take into account "spaces"

that's why I left one case with and one without 🤷‍♂️ 

if you don't think it's a good idea, I can change ;) 

See :
https://github.com/storybookjs/storybook/pull/13886/files#diff-5600a206076a8692aa20aa116acd9cb6ab00365444316e08aa5cb39d10050cebR27

File: examples/angular-cli/src/stories/addons/READEME.stories.mdx (hunk 1 above)
Comment (ThibaudAV) [reply examples/angular-cli/src/stories/addons/READEME.stories.mdx:1]: 🙄  🙇  :)

File: examples/angular-cli/src/stories/welcome-angular.stories.ts (hunk 5 above)
Comment (gaetanmaisse) [reply examples/angular-cli/src/stories/welcome-angular.stories.ts:6]: Ah! 👍🏻

**Review by shilman — APPROVED** [review]
💯 great job @ThibaudAV 😅

**Comment by shilman** [comment]
PS one deepscan change 😉 @ThibaudAV

**Comment by ThibaudAV** [comment]
✅  @shilman

**Pull request merged**